doc/ChangeLog

```diff
@@ -205,6 +205,9 @@ version 26-git (????-??-??):
     variables [chilan]
   - enhancement: promotions are in fact not deleted, but only marked as deleted [chilan]
   - improvement: improvement: auto-hide 'back to top' button [chilan]
+  - enhancement: chaning ticket queues resets ticket state to NEW
+    (if config variable: rt.ticket_queue_change_resets_ticket_state is set) [interduo]
```

Comment:
we wszystkich miescach w changelogu ustawienia umieszczamy w apostrofach i dla ułatwienia wpisujemy w nawiasie wartość domyślną?

Reply:
Ok - będę tak robił dodaj to do FAQ od razu, teraz przy każdym noworobionym PR sprawdzam po kolei punkt po punkcie

Reply:
Poprawione